susemanager/src/mgr_bootstrap_data.py

```diff
@@ -1516,6 +1535,11 @@
         'DEST' : DOCUMENT_ROOT + '/pub/repositories/ubuntu/20/4/bootstrap/',
         'TYPE' : 'deb'
     },
+    'ubuntu-22.04-amd64' : {
```

Comment:
@mcalmer I think this should go away for now, as we do not have the SCC client tools channels?